### PR TITLE
Fix/moving flow input validation

### PR DESCRIPTION
**Pull request by cpiehl1**
# Jira Issue: []

## What?
- Better error logging when moving flow input is invalid

## Why?
-

## Optional checklist
- [ ] Unit tests written
- [ ] Tested locally
- [ ] Codescouted

### CODE REVIEW

**Review by vonElfvin — APPROVED** [review]

File: src/resolvers/addressChange.ts [hunk 1]
```diff
@@ -260,3 +260,16 @@ const transformResult = (
...
+  (fieldName: string, value?: string): string
+  (fieldName: string, value?: boolean): boolean
+}
+
+const requiredField: IRequired = (fieldName: string, value?: any) => {
+  if (value == null) {
+    throw Error(`Field: ${fieldName} is required but was not provided`)
+  }
+  return value
+}
```
Comment (bystam) [inline src/resolvers/addressChange.ts:275]:
```suggestion
function requiredField<T>(fieldName: string, value: T | null | undefined) {
  if (!value) {
    throw Error(`Field: ${fieldName} is required but was not provided`)
  }
  return value
}
```

This simpler perhaps? :)

**Review by bystam — APPROVED** [review]
Looks good! Had one cosmetic suggestion that I think works :P

File: src/resolvers/addressChange.ts (hunk 1 above)
Comment (cpiehl1) [reply src/resolvers/addressChange.ts:275]: Yes! I wanted to do something like this but didn't get it to work 😄

Comment (cpiehl1) [reply src/resolvers/addressChange.ts]: Maybe because I was using an arrow function?

Comment (cpiehl1) [reply src/resolvers/addressChange.ts]: But I have to use `value == null`, otherwise it fails when `value = false`

**Pull request merged**